lib/ok.ex

```diff
@@ -205,13 +222,16 @@ defmodule OK do
 
   require Logger
 
-  @doc false
```

Comment:
Good call. I'm thinking that there are not likely to be many more breaking changes and we should do a 2.0 to get rid of this